docs/conceptual/runtime/srml.md

```diff
@@ -89,199 +89,199 @@ authorities, learn its own authority id, as well as to sign and verify messages
 voting.
 
 * [Docs](/rustdocs/master/srml_democracy/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/democracy/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/democracy/src/lib.rs)
 
 ### Elections Phragmen
 
 The Phragmen Elections module is an election module based on [sequential
 phragmen](https://research.web3.foundation/en/latest/polkadot/NPoS/4.%20Sequential%20Phragm%C3%A9n%E2%80%99s%20method/).
 
 * [Docs](/rustdocs/master/srml_elections_phragmen/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/elections-phragmen/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/elections-phragmen/src/lib.rs)
 
 ### Elections
 
 The Elections module is an election module for stake-weighted membership selection of a collective.
 
 * [Docs](/rustdocs/master/srml_elections/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/elections/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/elections/src/lib.rs)
 
 ### EVM
 
 The EVM Module is an [Ethereum](https://en.wikipedia.org/wiki/Ethereum) virtual machine (EVM)
 execution module for Substrate.
 
 * [Docs](/rustdocs/master/srml_evm/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/evm/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/evm/src/lib.rs)
 
 ### Example
 
 The Example module is a simple example of a runtime module demonstrating concepts, APIs and
 structures common to most runtime modules.
 
 * [Docs](/rustdocs/master/srml_example/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/example/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/example/src/lib.rs)
 
 ### Finality Tracker
 
 The Finality Tracker module tracks the last finalized block, as perceived by block authors.
 
 * [Docs](/rustdocs/master/srml_finality_tracker/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/finality-tracker/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/finality-tracker/src/lib.rs)
 
 ### Generic Asset
 
 The Generic Asset module provides functionality for handling accounts and asset balances.
 
 * [Docs](/rustdocs/master/srml_generic_asset/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/generic-asset/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/generic-asset/src/lib.rs)
 
 ### GRANDPA
 
 The GRANDPA module extends GRANDPA consensus by managing the GRANDPA authority set ready for the
 native code.
 
 * [Docs](/rustdocs/master/srml_grandpa/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/grandpa/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/grandpa/src/lib.rs)
```

Comment:
```suggestion
* [Source](https://github.com/paritytech/substrate/blob/master/palette/grandpa/src/lib.rs)
```
When a module belongs to the core is `palette` not `pallete`.

Reply:
ah, sorry, probably many of these errors to come

Reply:
updated

Reply:
So I think that replacing should work but it need to be `palette`
The differentiation is on the package name.

Reply:
Ex.
`pallet-balances`
`pallet-contracts`

Reply:
`pallet-contracts`
`palette-support`